delfin/drivers/dell_emc/vmax/alert_handler.py

```diff
@@ -17,129 +17,102 @@
 from oslo_log import log
 
 from delfin import exception
+from delfin.common import constants
 from delfin.drivers.dell_emc.vmax import alert_mapper
 
 LOG = log.getLogger(__name__)
 
 
 class AlertHandler(object):
     """Alert handling functions for vmax driver"""
-    default_category = 'New'
-    default_event_type = 'unknown'
-    default_probable_cause = 'Not available'
-    default_severity = 'unknown'
-    default_me_category = 'storage-subsystem'
-    default_location = 'unknown'
 
-    # Mandatory attributes expected in alert info to proceed with model filling
-    mandatory_alert_attributes = ('storage_id', 'storage_name', 'vendor',
-                                  'model', 'connUnitName', 'emcAsyncEventCode')
+    # Translation of trap severity to alert model severity
+    SEVERITY_MAP = {"emergency": constants.Severity.FATAL,
+                    "alert": constants.Severity.CRITICAL,
```

Comment:
Why Alert and Critical in same bucket of CRITICAL? Alert can be some soft threshold but critical is hard, isn't it

Reply:
It is as per vmax severity mapping